staging/src/k8s.io/apiserver/pkg/admission/metrics/testutil_test.go

```diff
@@ -21,11 +21,111 @@ import (
 
 	"github.com/prometheus/client_golang/prometheus"
 	ptype "github.com/prometheus/client_model/go"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/utils/pointer"
 )
 
+func TestLabelsMatch(t *testing.T) {
+	type testcase struct {
+		name          string
+		metric        *ptype.Metric
+		labelFilter   map[string]string
+		expectedMatch bool
+	}
+
+	testcases := []testcase{
```

Comment:
```go
tests := []struct{
		name          string
		metric        *ptype.Metric
		labelFilter   map[string]string
		expectedMatch bool
} {
  ...
}
```
nit: There is no need to define a struct, and the code above is more common in kubernetes. Others LGTM, Good work!

Reply:
fixed, thanks!